test/features/document.feature

```diff
@@ -0,0 +1,12 @@
+@search @functional
+Feature: Set of scenarios to test functionality of media viewer when the document link is clicked
+
+  Background:
+    Given I have logged in
+
+  Scenario: create a case with document upload
+    Given a case type containing every field type exists
```

Comment:
Might be clearer for the reader to use 'Given a case type with a document upload filed exists' then copy the implementation for the step

Reply:
Done.